scripts/run_inference.py

```
@@ -66,13 +66,17 @@ def run_inference(
-            save=False,
-            verbose=False,
-        )
+        try:
+            results = model.predict(
+                source=str(image_path),
+                conf=conf_threshold,
+                iou=iou_threshold,
+                save=False,
+                verbose=False,
+            )
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)